### PR TITLE
fix: fix doc comments of execute and execute_at_address

### PR DESCRIPTION
**Pull request by Eikix**
## Pull request type

Please check the type of change your PR introduces:

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [x] Documentation content changes
- [ ] Other (please describe):

## What is the current behavior?

Resolves #<Issue number>

## What is the new behavior?

-
-
-

## Other information

Time spent on this PR:

### CODE REVIEW

**Comment by Flydexo** [comment]
Is it technically possible to make the execute_at_address also as a view function so we can call EVM view functions without triggering starknet transactions?

**Comment by Eikix** [comment]
> Is it technically possible to make the execute_at_address also as a view function so we can call EVM view functions without triggering starknet transactions?

Unfortunately no, view functions don't modify the state of the chain. Here, execute_at_address may write to storage and thus must be an external function.

File: src/kakarot/kakarot.cairo [hunk 1]
```diff
@@ -31,8 +31,9 @@ func constructor{syscall_ptr: felt*, pedersen_ptr: HashBuiltin*, range_check_ptr
 // @param bytecode The bytecode to be executed
 // @param calldata_len The calldata length
 // @param calldata The calldata which can be referenced by the bytecode
+// @return stack_accessess_len The size of the accesses array of the stack delta
```
Comment (ClementWalter) [inline src/kakarot/kakarot.cairo:34]: `accesses` (!)

**Review by ClementWalter — DISMISSED** [review]

File: src/kakarot/kakarot.cairo (hunk 1 above)
Comment (Eikix) [reply src/kakarot/kakarot.cairo]: done

File: src/kakarot/kakarot.cairo [hunk 2]
```diff
@@ -31,7 +31,7 @@ func constructor{syscall_ptr: felt*, pedersen_ptr: HashBuiltin*, range_check_ptr
 // @param bytecode The bytecode to be executed
 // @param calldata_len The calldata length
 // @param calldata The calldata which can be referenced by the bytecode
-// @return stack_accessess_len The size of the accesses array of the stack delta
+// @return stack_accesses_len The size of the accesses array of the stack delta
```
Comment (ClementWalter) [inline src/kakarot/kakarot.cairo:34]: Since you’re here, there’s `accessesses` down below

**Review by ClementWalter — APPROVED** [review]

File: src/kakarot/kakarot.cairo (hunk 2 above)
Comment (Eikix) [reply src/kakarot/kakarot.cairo:34]: done

**Review by ClementWalter — APPROVED** [review]

**Pull request merged**